src/pagination/pagination.service.js

```diff
@@ -0,0 +1,24 @@
+(function () {
```

Comment:
It appears that the pagination service isn't currently used by the pagination component? (And the setPages function isn't implemented in the service itself.) Perhaps we should omit this until we can show an example that works with table and list view, for example?